Makefile

```diff
@@ -119,6 +123,12 @@ html-lint:  ## Validate HTML in web application template files.
 		securedrop/source_templates/*.html securedrop/journalist_templates/*.html
 	@echo
 
+.PHONY: rust-lint
+rust-lint: ## Lint Rust code
```

Comment:
Two considerations:
1. We don't currently require a Rust toolchain to run `make lint/test`
2. I would estimate that 95% of commits (currently) won't touch Rust code, so running it on every lint/test seems excessive and if you don't have a hot cache, slow.

I expect both of those to degrade over time (the first is especially weak already) that it's probably worth doing eventually, it just seems like it would be a bit annoying right now.

Reply:
Makes sense.  Thanks!